plugins/woocommerce/client/legacy/js/frontend/add-to-cart.js

```diff
@@ -139,9 +153,7 @@ jQuery( function( $ ) {
 		e.data.addToCartHandler.addRequest({
 			type: 'POST',
 			url: wc_add_to_cart_params.wc_ajax_url.toString().replace( '%%endpoint%%', 'remove_from_cart' ),
-			data: {
-				cart_item_key : $thisbutton.data( 'cart_item_key' )
-			},
+			data: 'cart_item_key=' + encodeURIComponent( $thisbutton.data( 'cart_item_key' ) ),
```

Comment:
we can change this string concatenation to template strings or we could use `URLSearchParams`. I probably prefer params just because it also handles encoding.

```suggestion
			data: new URLSearchParams({ cart_item_key:  $thisbutton.data( 'cart_item_key' ) }).toString(),
```

Reply:
Yes, should definitely use URLSearchParams() here, I'll commit a change.